install/WindowsNew.shtml

```diff
@@ -140,6 +140,19 @@ For earlier versions (2012 and before), <a href="Windows.shtml">refer to the old
 <a id="installation"></a>
 <h2>Installation Procedure</h2>
 <ol>
+  <li><p>Since JMRI 5.1.6. During the installation your machine will be checked for java 11 or greater.
+      First it will check the default version, then if that fails search the registry entries (javasoft keys).</p>
+      <p>If java 11 or greater is not found it will ask you whether you wish to continue the installation.
+      Continuing permits you to resolve this issue later by either installing a valid version or linking JMRI
+      to an absolute version as described in <a href="#specificjavaversion">Using a specific java version</a> section below.</p>
+
+      When running JMRI the search order for java (specifically bin/javaw.exe) is 1 - the jre directory, 2 - the default java
+      and 3 - the first valid version 11 or higher found in the registry keys.
```

Comment:
I suggest that you have an ordered list for these three items. It makes it easier to read.

Reply:
Done